Clinical-Genomics/scout
# BUGFIX - Optional institute-specific Alamut Visual Plus API key in Alamut links

Fix #2745. Allows providing an institute-specific Alamut Visual Plus key if this software is present.

In that case the link to alamut from variants page will be the following:
http://localhost:10000/search?apikey=blablabla&request=7:124491972C%3EA

If no alamut key is set for the given institute then the link to Alamut will be this:
http://localhost:10000/show?request=7:124491972C%3EA


**How to test**:
1. Install on clinical-db stage
2. As an admin user, edit institute settings for an institute who has access to Alamut:
![image](https://user-images.githubusercontent.com/28093618/128828311-1aa04d0e-9139-449d-95d6-e91bce1b9d5a.png)
3. Go to a variant and click on the Alamut link, check that link works
4. Try to reset the Alamut link to None from the institute settings by providing a blank string for this paremeter

**Expected outcome**:
The alamut link should contain `apikey` args 

**Review:**
